regression/cbmc-java/static_init_order/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+static_init_order.class
+
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
I think one positive test is not enough to show the feature works. There should be at least one test where an assertion can fail.